openid-connect-server-webapp/src/main/resources/entity-mappings_oracle.xml

```diff
@@ -0,0 +1,266 @@
+<entity-mappings xmlns="http://xmlns.jcp.org/xml/ns/persistence/orm"
```

Comment:
I used this file to change values provided in java annotations for entities - that way we do not need to modify the code to get it working for oracle. Two types of changes here:
* sequence generator insteqad of identity for IDs
* identifiers exceeding 30 characters shortened